src/ensembl/src/shared/components/question-button/QuestionButton.tsx

```diff
@@ -14,10 +14,10 @@
  * limitations under the License.
  */
 
-import React, { useState, useEffect } from 'react';
+import React from 'react';
 import classNames from 'classnames';
 
-import useHover from 'src/shared/hooks/useHover';
+import { useQuestionButtonBehavior } from './useQuestionButtonBehavior';
```

Comment:
Aren't we sticking with UK spellings for variables?

Reply:
we may need to change this too https://github.com/Ensembl/ensembl-client/blob/dev/src/ensembl/src/content/app/entity-viewer/gene-view/components/proteins-list/proteins-list-item/ProteinsListItem.tsx#L102 😆

Reply:
I just did a quick check in the code and found we are sticking on to British English. Sorry @azangru  :-)

Reply:
We can't change that, Jyo; it's the browser API, and browsers speak American English:

https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView

This is why I would love us to use American English in code. All the other examples of the word `behaviour` in our code are for display labels; which is totally fine, we could write labels in Chinese hyerogliphs for all we care. But the code itself would be much more consistent if it stuck to American spelling 😞

Reply:
oh oh!

Reply:
It is with heavy heart that I have changed behavior for behaviour 😞 

✅